### PR TITLE
Feature EPL-1728: First model of the Tab container component

### PR DESCRIPTION
**Pull request by Sankku**
First model of the Tab Navigation

image

### CODE REVIEW

File: packages/ComponentLibrary/src/components/Table/TabNavigation/TabContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import TabContent from './TabContent';
+import type { TabProps } from '../../../../../storybook/src/stories/Components/Table/types';
```
Comment (LuuchoRocha) [inline packages/ComponentLibrary/src/components/Table/TabNavigation/TabContainer.tsx:2]: hmmm this doesn't look good.
As far as I can understand, the `TabProps` type should be part of the ComponentLibrary

File: packages/ComponentLibrary/src/components/Table/TabNavigation/TabContent.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { Box, Typography } from '@mui/material';
+import { TabContentProps } from '../../../../../storybook/src/stories/Components/Table/types';
```
Comment (LuuchoRocha) [inline packages/ComponentLibrary/src/components/Table/TabNavigation/TabContent.tsx:2]: hmmm this doesn't look good.
As far as I can understand, the `TabContentProps` type should be part of the ComponentLibrary

File: packages/ComponentLibrary/src/components/Table/TabNavigation/index.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
+import { useState, useRef, useCallback, useEffect } from 'react';
+import { Box, Paper } from '@mui/material';
+import TabContainer from './TabContainer';
+import { resizableTabContainerStyles as styles } from '../styles';
+import type { ResizableTabContainerProps } from '../../../../../storybook/src/stories/Components/Table/types';
```
Comment (LuuchoRocha) [inline packages/ComponentLibrary/src/components/Table/TabNavigation/index.tsx:5]: same

**Review by LuuchoRocha — CHANGES_REQUESTED** [review]
Great work buddy!
Please address the two minor issuesa about type definitions and I'll approve this one 👍🏼

**Review by LuuchoRocha — APPROVED** [review]

**Review by sjcuello — APPROVED** [review]

**Pull request merged**